src/utils/analytics-get-player-options.js

```diff
@@ -0,0 +1,45 @@
+// eslint-disable-next-line complexity
+export const analyticsGetPlayerOptions = (playerOptions) => {
+  return {
+    showJumpControls: playerOptions.showJumpControls || null,
+    seekThumbnails: playerOptions.seekThumbnails || null,
+    aiHighlightsGraph: playerOptions.aiHighlightsGraph || null,
+    floatingWhenNotVisible: playerOptions.floatingWhenNotVisible || null,
+    hideContextMenu: playerOptions.hideContextMenu || null,
+    analytics: playerOptions.analytics || null,
+    cloudinaryAnalytics: playerOptions.cloudinaryAnalytics || null,
+    playedEventPercents: playerOptions.playedEventPercents || null,
+    ...(playerOptions.cloudinary ? {
+      sourceTypes: playerOptions.cloudinary.sourceTypes || null,
+      autoShowRecommendations: playerOptions.cloudinary.autoShowRecommendations || null,
+      transformation: playerOptions.cloudinary.transformation || null,
+      fontFace: playerOptions.cloudinary.fontFace || null,
+      ...(playerOptions.cloudinary.posterOptions ? {
+        posterOptionsPublicId: playerOptions.cloudinary.posterOptions.publicId || null,
```

Comment:
[important] We do not want to send publicIds, so please here if this exists please send true otherwise false. (This way we can understand if customers used specific posters or not)